patches/third_party/devtools-frontend/src/front_end-panels-network-NetworkItemView.ts.patch

```diff
@@ -0,0 +1,9 @@
+diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
+index 645aae8d1fb76a898cea88baf3f39ef5675449cb..3825908d06094dde919796af8a6827d8db173d51 100644
+--- a/front_end/panels/network/NetworkItemView.ts
++++ b/front_end/panels/network/NetworkItemView.ts
+@@ -359,3 +359,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
+     return this.headersViewComponent;
+   }
+ }
++import { NetworkItemViewPatch } from './NetworkItemView.patch.js'; (NetworkItemView as any) = NetworkItemViewPatch(NetworkItemView);
```

Comment:
`NetworkItemViewPatch` -> `patchNetworkItemView`? same for other patch functions.